samples/tfm_integration/tfm_regression_test/prj.conf

```diff
@@ -8,3 +8,6 @@ CONFIG_TEST=y
 CONFIG_BUILD_WITH_TFM=y
 CONFIG_TFM_REGRESSION_S=y
 CONFIG_TFM_REGRESSION_NS=y
+
+# TF-M Regression tests expect SHA-1 to be disabled
+CONFIG_MBEDTLS_SHA1_C=n
```

Comment:
This is introducing a downstream symbol in sdk-zephyr which should normally be avoided.

This symbol should default to `n` from here:
https://github.com/nrfconnect/sdk-nrf/blob/4aa68fe9c49604ca66b96d4333dc9dbc3362b734/modules/trusted-firmware-m/Kconfig.mbedtls.defconfig#L7-L8

so it should be investigated why this is not working as expected instead of introducing yet another layer using a temphack.

Reply:
I see https://github.com/nrfconnect/sdk-nrf/blob/4aa68fe9c49604ca66b96d4333dc9dbc3362b734/modules/trusted-firmware-m/Kconfig.mbedtls.defconfig is never being sourced in TF-M Kconfig.
@hakonfam I assume that's a mistake.

Reply:
Took me some time to realize what was failing, but I started working on fixing the failing tests in a different way. Are the mbdetls symbols expected to be available in sdk-zephyr, or is the changes in this PR a dead-end that will never work?